src/components/my_meeting/HomeMyMeetings.js

```diff
@@ -1,15 +1,38 @@
 "use-strict";
 import React, { Component } from "react";
-import { View, Text } from "react-native";
+import { View, StyleSheet, FlatList, Dimensions } from "react-native";
+import { CRHeader } from "src/components/common"
+import MeetingCell from "src/components/my_meeting/MeetingCell"
+
+const cellWidth = Dimensions.get("window").width;
+const data = [{ id: "0", title: "hola" }, { id: "1", title: "Hola" }, { id: "2", title: "Hola" }, { id: "3", title: "Hola" }];
 
 class HomeMyMeetings extends Component {
```

Comment:
favor functional components over classes

Reply:
This is a screen, is it ok to convert it into a functional component?

Reply:
yeah, convert it

Reply:
done